charts/kong-operator/templates/NOTES.txt

```diff
@@ -5,4 +5,4 @@
 For more details, please refer to the following documents:
 
 * https://docs.konghq.com/gateway-operator/latest/get-started/kic/create-gateway/
-* https://docs.konghq.com/gateway-operator/latest/get-started/konnect/deploy-data-plane/
+* https://docs.konghq.com/konnect/gateway-manager/data-plane-nodes/
```

Comment:
This link should rather be

```suggestion
* https://docs.konghq.com/gateway-operator/latest/get-started/konnect/deploy-data-plane-with-konnectextension/
```

The reason for this is in 1.5 the link slug changed (in 1.4 it has been: https://docs.konghq.com/gateway-operator/1.4.x/get-started/konnect/deploy-data-plane/)